packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -61,6 +61,23 @@ export interface LoadBalancedServiceBaseProps {
    * @default - No environment variables.
    */
   readonly environment?: { [key: string]: string };
+
+  /**
+   * Properties of the ELBv2 TargetGroup
+   *
+   * Can be used to override the default healthCheck, for example:
+   * ```ts
+   * targetProps: {
+   *   port: 80
+   *   healthCheck: {
+   *     path: "/health"
+   *   }
+   * }
+   * ```
+   *
+   * @default { port: 80 }
+   */
+  readonly targetProps?: elbv2.AddApplicationTargetsProps & elbv2.AddNetworkTargetsProps;
```

Comment:
This is not supported by `jsii`. You will have to make the union type of these 2 yourself. Could be easy enough doing something like:

```ts
export interface TargetProps extends AddApplicationTargetsProps, AddNetworkTargetsProps {
}
```

The linter will complain but you can silence it.

Needs some validation on not using inappropriate fields viz-a-viz the LB type though.